examples/resolution/README.md

```diff
@@ -0,0 +1,131 @@
+# Gatsby Resolution Example
+
+[![Netlify Status](https://api.netlify.com/api/v1/badges/2f5f005f-1c66-4022-8c78-382f0a06a2a7/deploy-status)](https://app.netlify.com/sites/kontent-gatsby-example-resolution/deploys)
+
+Site showcasing how to resolve Rich Text element and it's inline images, incline content items, and content components.
+
+## Get started
+
+:warning: Before running any of the following scripts locally ensure the site is using the latest dependencies
+
+```sh
+# In the root folder of this repo
+yarn # install all required packages
+yarn build # build the latest version of the local packages
+```
+
+### Develop site
+
+```sh
+# open the /example/navigation folder
+cd example/navigation
+yarn develop # runs `gatsby develop` command
+```
+
+Now you could browse the site on <http://localhost:8000> and see GraphiQL explorer on <http://localhost:8000/___graphql>.
+
+## Content modeling
+
+This section explains how the content is modeled. You could follow next section of [Import the content to your on Kontent project](#Import-site-content-to-your-Kontent-project) and explore the models by your own as well as the sample data based on it.
+
+Once you create the content models, you could create content items based on these and the site would be capable to handle the content and render it.
+
+### Kontent content models
+
+First of of all it is required to create two content types:
+
+- `Person` - which will represent a person on our website and mainly `Bio` rich text element used for resolution showcase
+- `Repository` - this content type represents a github repository information - used as an inline lik item from Person's `Bio` element
+- `Website` - this content type represents a website information - used as an inline lik item from Person `Bio` element
+
+#### Person content modeling
+
+Following section describes ths content structure (content model). If you want, you could 
+
+This content type - `Person` - has following structure:
+
+- Name - **Text** element
+- Bio - **Rich text** element - configures to allow to link only `Repository` and `Website` content types
+- Slug - **URL slug** element - generated from name
+
+This content type represents people with their `Bio`. They could link any website, or repository to their bio and these components needs to be resolved.
```

Comment:
Ntt sure what `This` context is - it does not make sense to me

Reply:
f4494b1